command/login.go

```diff
@@ -234,7 +234,11 @@ func (c *LoginCommand) Run(args []string) int {
 			// request is validated interactively
 			methodInfo := c.getMFAMethodInfo(secret.Auth.MFARequirement.MFAConstraints)
 			if methodInfo.methodID != "" {
-				return c.validateMFA(secret.Auth.MFARequirement.MFARequestID, methodInfo)
+				secret, err = c.validateMFA(secret.Auth.MFARequirement.MFARequestID, methodInfo)
```

Comment:
I need to examine this a bit closer and make sure it's doing the correct thing. Open to any/all feedback in the meantime.

Reply:
Should we return here ? The warning log after that should not be printed if this validate works. Not sure if I follow what is the ultimate goal here though. Is it that we only need to store the token in the token helper?